Cargo.toml

```
@@ -21,4 +21,7 @@ tempfile = "3"
 toml = "0.5"
 
 [target.'cfg(target_os = "windows")'.dependencies]
-bindings = { path = "bindings" }
+cargo-temp-bindings = { path = "cargo-temp-bindings" }
+
+[workspace]
+members = ['cargo-temp-bindings']
```

```suggestion
members = ["cargo-temp-bindings"]
```
Try to keep consistency